flink-formats/flink-orc/src/main/java/org/apache/flink/orc/vector/OrcTimestampColumnVector.java

```diff
@@ -39,8 +41,8 @@ public OrcTimestampColumnVector(TimestampColumnVector vector) {
 	@Override
 	public SqlTimestamp getTimestamp(int i, int precision) {
 		int index = vector.isRepeating ? 0 : i;
-		return SqlTimestamp.fromEpochMillis(
-				vector.time[index],
-				SqlTimestamp.isCompact(precision) ? 0 : vector.nanos[index] % 1_000_000);
+		Timestamp timestamp = new Timestamp(vector.time[index]);
```

Comment:
What's the difference with the original one?

Reply:
Original one directly use the underlying `long` and `int` to construct `SqlTimestamp`.
But hive orc is using `java.sql.Timestamp` to construct underlying data. You can understand like:
```
java.sql.Timestamp orcTimestamp;
SqlTimestamp.fromEpochMillis(orcTimestamp.getTime(), orcTimestamp.getNano());
VS
SqlTimestamp.fromString(orcTimestamp.toString());
```

Reply:
So the later one will be influenced by local time zone?

Reply:
Yes